starcli/search.py

```diff
@@ -27,26 +29,44 @@
     "valid": "The request returned successfully, but an unknown exception occurred.",
 }
 
+FORMAT = "%(message)s"
+
+
+
+httpclient_logger = logging.getLogger("http.client")
+
+def httpclient_logging_debug(level=logging.DEBUG, debug_level=1):
+
+    def httpclient_log(*args):
+        httpclient_logger.log(level, " ".join(args))
+
+    http.client.print = httpclient_log
+    http.client.HTTPConnection.debuglevel = 1
+
 
 def debug_requests_on():
     """ Turn on the logging for requests """
+
+    logging.basicConfig(
+        level=logging.DEBUG, format=FORMAT, datefmt="[%Y-%m-%d]", handlers=[RichHandler()]
+    )
     logger = logging.getLogger(__name__)
+
     try:
         from http.client import HTTPConnection
+        httpclient_logging_debug(debug_level=1)
 
-        HTTPConnection.set_debuglevel(HTTPConnection, 1)
     except ImportError:
         import httplib
 
-        httplib.HTTPConnection.debuglevel = 2
+        httpclient_logging_debug(debug_level=2)
```

Comment:
The try-except is not needed here anymore, because i don't think `from http.client import HTTPConnection` is used, also why is `debug_level` set to 2 in the except block, even though the debug level is set to 1 in the function (see my comment above)?

Reply:
I had kept the try except block as a safeguard, but I understand how it is not necessary. I am resolving it in the new commit.